dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -1,64 +1,117 @@
 #!/usr/bin/env python
 
 import sys
+import copy
+import Queue
 import redis
 import subprocess
 import syslog
-from swsssdk import ConfigDBConnector
+import os
+from swsscommon import swsscommon
 
-class BGPConfigDaemon:
 
-    def __init__(self):
-        self.config_db = ConfigDBConnector()
-        self.config_db.connect()
-        self.bgp_asn = self.config_db.get_entry('DEVICE_METADATA', 'localhost')['bgp_asn']
-        self.bgp_neighbor = self.config_db.get_table('BGP_NEIGHBOR')
+def run_command(command):
+    syslog.syslog(syslog.LOG_DEBUG, "[bgp cfgd] execute command {}.".format(command))
```

Comment:
You don't need to add the process name into the syslog messages. If you first call `syslog.openlog("bgpcfgd")`, this identifier will be added to the log message. You can call it once at startup and call `syslog.closelog()` at shutdown, or you can follow some of our other applications which create log wrappers: https://github.com/Azure/sonic-utilities/blob/master/psuutil/main.py#L41

Reply:
Thanks for your suggestion, I have updated it.